Start of page highlighting GitHub discussion showing screenshots

stumbo opened the pull request:
Start of a page that points to the discussion group.  Need to put some more thought into the wording, but the mechanics are there.

stumbo committed:
Start of page highlighting GitHub discussion showing screenshots

pamoroso commented on content/en/software/screenshots/_index.md:
```diff
@@ -0,0 +1,14 @@
+description:  A link to GitHub discussion page featuring Medley screenshots
+--- 
+
+## Medley in use
```
I'd change the heading to something like "Screenshots of Medley in use" or "Showcasing Medley in use" to reinforce that the section is about visuals.

stumbo replied:
I'll update it to **Showcasing Medley in use** .

masinter commented:
is there a way to review wihtout merging?

stumbo committed:
Removed description, fixed typo and shrank the size of the image

Description text prints between the title and first header at
a larger size.  Between that and the size of the image the title
was pushed off the screen.

stumbo commented:
Here's a picture of what it currently looks like:
![image](https://github.com/user-attachments/assets/05351dae-42fa-45bd-b27f-2fa68d124655)

I can load it on the staging server - but that'll take a little more time than I have available right now.

masinter commented:
The problem with the screenshots as a "Discussion" is that it isn't really set up to "discuss" the screenshots. You don't want people to discuss a screenshot in the same "discussion" -- but there's no other obvious choice for where to discuss.

I can think of some alternatives:

- Screenshots in GitBook
- re-open medley wiki
- a screenshot repository
- Another Discussion for "Discuss Screenshots" 
- ...

Things I'd like to know about the screenshots:
* related applications/screenshots
* questions or instructions on how to recreate
* related videos

stumbo commented:
I agree this might not be the right direction.  Maybe this is something better suited for our social media sites?  We could use some of the material in the Discussion topic to seed the conversation.  

If we feel strongly we want to put it on the website, we should probably devote a single page to each screenshot and put some more material around the screenshot - focusing on some of the questions you asked.  Which sounds like a tall order right now.  I also think moving to GitBook or the GitHub wiki is going to a difficult and getting eyeballs there will also be a challenge.

pamoroso commented:
What about rephrasing like this to deemphasize discussion? The sparse text in the discussion itsels is another hint.

> Medley provides a rich palette for displaying information.  We maintain an extensive screenshot gallery allowing users to post images of their screen, highlighting the flexibility of the Medley interface and the range of usages of the system.  We invite you to browse our collection and add to it.
> 
> Screenshot gallery of Medley in use

I originally opened the discussion as a lightweight sharing space with a focus on visuals rather than technical details (but the Exec in most of the screenshots shows the Lisp expressions to bring up the showcased programs or tools). However, we may migrate to a new resource with additional features or metadata.

stumbo committed:
Update page to better hightlight the purpose of the GitHub Discussion

stumbo commented:
Updated the text based on the discussion at the 20250709 External Group meeting.  Here's what the page currently looks like:
![image](https://github.com/user-attachments/assets/1b7efff9-17d6-4c61-ba82-d23a0f5084df)

stumbo committed:
Merge branch 'main' into postYourScreenshot

pamoroso reviewed (approved):
Looks good to me.

stumbo committed:
Update heading based on Paolo's feedback
